### ISSUE TITLE
npm module aion-web3-core

### ISSUE THREAD

**Issue by cybercoder2009** [aionnetwork/aion_web3#34]
os: ubuntu
node version: 8.9.1
npm: 5.5.1 
----------------------------------------------------------------------------
try 
```
var Web3 = require('aion-web3-core')
var web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
```
which gives
```
module.js:538
    throw err;
    ^

Error: Cannot find module 'lodash/padStart'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/chris/test-aion-web3-core/src/node_modules/aion-lib/src/abi.js:6:16)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
```

**Comment by aion-kelvin** [comment]
Thanks for the report.  We need to update aion-web3's dependencies to require that module.  

In the meanwhile, a temporary workaround is to run `npm i lodash.padstart` in that directory to manually install the dependency.

**Comment by aion-kelvin** [comment]
fixed by #41
